shoop/core/models/products.py

```diff
@@ -535,6 +535,10 @@ def get_all_package_children(self):
             ProductPackageLink.objects.filter(parent=self).values_list("child", flat=True)
         ))
 
+    def get_public_media(self):
+        from .product_media import ProductMedia
+        return ProductMedia.objects.filter(product=self, enabled=True, public=True)
```

Comment:
The custom manager stuff is out of the scope of this issue though, so do the first one...